julia_src/Manifest.toml

```diff
@@ -713,9 +713,11 @@ uuid = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 
 [[deps.REopt]]
 deps = ["ArchGDAL", "CSV", "CoolProp", "DataFrames", "Dates", "DelimitedFiles", "HTTP", "JLD", "JSON", "JuMP", "LinDistFlow", "LinearAlgebra", "Logging", "MathOptInterface", "Requires", "Roots", "Statistics", "TestEnv"]
-git-tree-sha1 = "f0bb76959fb18989d9cc8510a9fd06e37564c3bf"
+git-tree-sha1 = "0d9bdc5ead9c9837146505bea1b4981432aa18ef"
+repo-rev = "wind-erp"
+repo-url = "https://github.com/NREL/REopt.jl.git"
 uuid = "d36ad4e8-d74a-4f7a-ace1-eaea049febf6"
-version = "0.32.7"
+version = "0.32.6"
```

Comment:
Off-topic comment - why does it go back like this? It did the same thing for me when I used a feature branch of REopt.jl which was branched off of a 0.32.7 version.

Reply:
I'd guess someone updated the version # in project.toml directly in master after merging their PR instead of in develop before merging. If so, easily resolved by merging master back into develop.